ktor-network/ktor-network-quic/jvmAndNix/src/io/ktor/network/quic/streams/QUICStreamChannel.kt

```diff
@@ -7,23 +7,14 @@ package io.ktor.network.quic.streams
 import io.ktor.network.sockets.*
 import kotlinx.coroutines.channels.*
 
-public typealias QUICStream = ByteArray
-
 /**
  * A channel for sending QUIC streams
  */
 public interface QUICStreamWriteChannel {
     /**
-     * QUIC stream outgoing channel.
-     */
-    public val outgoing: SendChannel<QUICStream>
-
-    /**
-     * Send QUIC stream.
+     * Create new QUIC stream.
      */
-    public suspend fun send(stream: QUICStream) {
-        outgoing.send(stream)
-    }
+    public suspend fun createStream(): QUICStream
```

Comment:
there should be possibility to open both unidirectional and bidirectional streams

Reply:
Yes, for now it is only bidirectional to make PoC implementation (in current state in works for basic cases from the start to the end). I plan to follow the spec more precisely in next iterations, including support for both uni- and bi- directional streams